demisto_sdk/commands/content_graph/parsers/xsiam_dashboard.py

```diff
@@ -16,28 +15,32 @@ class XSIAMDashboardParser(
     JSONContentItemParser, content_type=ContentType.XSIAM_DASHBOARD
 ):
     def __init__(
-        self, path: Path, pack_marketplaces: List[MarketplaceVersions]
+        self,
+        path: Path,
+        pack_marketplaces: List[MarketplaceVersions],
+        git_sha: Optional[str] = None,
     ) -> None:
-        super().__init__(path, pack_marketplaces)
-        self.json_data: Dict[str, Any] = self.json_data.get("dashboards_data", [{}])[0]
+        super().__init__(path, pack_marketplaces, git_sha=git_sha)
 
-    @property
-    def supported_marketplaces(self) -> Set[MarketplaceVersions]:
-        return {MarketplaceVersions.MarketplaceV2}
-
-    @property
-    def object_id(self) -> Optional[str]:
-        return self.json_data.get("global_id")
-
-    @property
-    def fromversion(self) -> str:
-        return (
-            self.original_json_data.get("fromVersion")
-            or DEFAULT_CONTENT_ITEM_FROM_VERSION
+    @cached_property
+    def mapping(self):
+        super().mapping.update(
+            {
+                "object_id": [
+                    "dashboards_data[0].global_id",
+                    "dashboards_data[0].id",
+                    "id",
+                    "global_id",
```

Comment:
?

Reply:
there are multiple options for where this key might be.. get_value will know how to find the right one.

Reply:
can you please share a path to an existing XSIAM dashboard that has "id" or "global_id" fields in the root level of the json?

Reply:
Oh my bad, looking into this again, I got confused because in the super class we'd extract dashboards_data and then call id / global_id.. removing.